src/main/java/nl/novi/behuiswerkles11/controllers/TelevisionsController.java

```diff
@@ -36,4 +36,4 @@ public ResponseEntity<Object> deleteTelevision (@PathVariable int id) {
         return ResponseEntity.noContent().build();
     }
 
-}
+}
```

Comment:
Je PR is niet helemaal goed, dus vandaar het commentaar hier en niet bij de jusite regel. 
Je gebruikt de juiste mapping (Http methodes) en je returned de juiste statussen.
Goed dat je hier ook de specifiekere Generic gebruikt en niet gewoon <Object>.
<Object> kun je wel gebruiken, maar alleen als het niet anders kan.